airbyte-workers/src/main/java/io/airbyte/workers/internal/AirbyteMessageTracker.java

```diff
@@ -386,4 +393,17 @@ protected Long calculateMean(final Long currentMean, final Long totalCount, fina
     return (long) result;
   }
 
+  private void logMessageAsJSON(final String caller, AirbyteMessage message) {
+    if (!logConnectorMessages) {
+      return;
+    }
+
+    try {
+      final String json = new ObjectMapper().writeValueAsString(message);
```

Comment:
I think we have [a helper for that](https://github.com/airbytehq/airbyte/blob/e9afa9bef3fcd5bf4237bea1295973ccd0c1f1e5/airbyte-commons/src/main/java/io/airbyte/commons/json/Jsons.java#L44)

Reply:
Nice! added via https://github.com/airbytehq/airbyte/pull/15895/commits/3b0006f7347a7f6115a08d0784fc64345ae2bbc7